### ISSUE TITLE
Failed to add Presentation

### ISSUE THREAD

**Issue by alanclayton** [Rise-Vision/rise-vision-apps#803]
Hannah and Meg both receive the error "Failed to add Presentation" after performing the steps below.

1. go to sub company Rise Tradeshow
2. Presentations > Add Presentation > Blank 
3. Save Presentation

After step 3 they get the following error:

> Failed to add Presentation. The Presentation could not be added. Field "isStoreProduct' is not editable. 

https://www.screencast.com/t/5d3MBJladA

I do not get this error when I perform the steps from my account.

**Comment by ezequielc** [comment]
@alanclayton A quick solution is to add Rise Store Admin role to them. 
We identified this issue in the past. It only happens internally, when we set a Rise Admin role to a new account but don't set Rise Store Admin role. 
At that time, we decided it was not a priority to fix given the simple workaround, but I will check the effort now to see if it is worth pursuing it. Thanks

**Comment by alanclayton** [comment]
@ezequielc so this only happens to somebody that belongs to the Rise Vision parent company?

**Comment by ezequielc** [comment]
Yes, correct.

Em qui, 17 de jan de 2019 20:58, Alan Clayton <notifications@github.com
escreveu:

**Comment by alex-deaconu** [comment]
As per the fact that this happens only in the Rise Vision Company and to new Users that are not configured with the correct permissions, we are going to close the issue and address at a later date. Thanks.
